src/components/repository-list-item.component.js

```diff
@@ -108,5 +108,6 @@ export const RepositoryListItem = ({ repository, navigation }: Props) =>
       type: 'octicon',
     }}
     underlayColor={colors.greyLight}
-    onPress={() => navigation.navigate('Repository', { repository })}
+    onPress={() =>
+      navigation.navigate('Repository', { name: repository.full_name })}
```

Comment:
Most of our screens can be passed either a "string" or an "object" for the principal entity they're dealing with.
For example, user-profile can either take a `login` name, or a `user` object. This is really confusing and generate a lot of checks.

By switching to normalizer, we can now pass only the login, and let the middleware fetch the data from store or api :)